packages/elements/src/components/ino-datepicker/ino-datepicker.tsx

```diff
@@ -58,9 +58,14 @@ export class Datepicker implements ComponentInterface {
 
   @Watch('value')
   valueChanged(value: string) {
-    this.setValidState(value);
-    if (this.flatpickr) {
-      this.flatpickr.setDate(value, false, this.inoDateFormat);
+    try {
+      if (this.flatpickr) {
+        this.setValidState(value);
+        this.flatpickr.setDate(value, false, this.inoDateFormat)
```

Comment:
With this logic, the flatpickr will always set the date, even if it is invalid. Is this intended?